app/src/main/java/no/usn/mob3000/ui/screens/user/ProfileEditScreen.kt

```diff
@@ -93,7 +95,8 @@ fun ProfileEditScreen(
                 value = aboutMe,
                 onValueChange = { aboutMe = it },
                 label = { Text(stringResource(R.string.profile_edit_about_me)) },
-                placeholder = { Text(stringResource(R.string.profile_edit_about_me_placeholder)) },
+                placeholder = { Text(stringResource(R.string.profile_edit_about_me_placeholder), color = MaterialTheme.colorScheme.onBackground.copy(alpha = 0.6f)) },
+                textStyle = TextStyle(color = MaterialTheme.colorScheme.onBackground),
```

Comment:
[See this comment](https://github.com/frigvid/mob3000-group11/pull/27/files#r1847123849).